app/src/main/java/com/example/spendwithbrain/db/dao/UserDAO.kt

```diff
@@ -1,12 +1,12 @@
-package com.example.spendwithbrain.db.interfaces
+package com.example.spendwithbrain.db.dao
 
-import androidx.room.Dao
-import androidx.room.Insert
-import androidx.room.OnConflictStrategy
-import androidx.room.Query
+import androidx.room.*
 import com.example.spendwithbrain.db.entities.ExpensesDetails
 import com.example.spendwithbrain.db.entities.IncomeDetails
 import com.example.spendwithbrain.db.entities.UserDetails
+import com.example.spendwithbrain.db.models.UserWithExpenses
+import io.reactivex.Flowable
+import io.reactivex.Single
```

Comment:
I don't think you use this imports: io.reactivex.Flowable/Single, please remove them